### PR TITLE
Dockerfile added

### PR DESCRIPTION
**Pull request by denysvitali**
I've added a Dockerfile to quickly deploy a box.js instance. Enjoy!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM mhart/alpine-node:latest
```
Comment (CapacitorSet) [inline Dockerfile]: There are official Node images built on Alpine: `FROM node:alpine`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+RUN apk add --no-cache git file
+RUN git clone https://github.com/CapacitorSet/box-js /usr/share/boxjs;\
```
Comment (CapacitorSet) [inline Dockerfile]: Why do this in the same `RUN` command?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+cd /usr/share/boxjs; npm install;
```
Comment (CapacitorSet) [inline Dockerfile]: `WORKDIR /usr/share/boxjs`; also, use `npm install --production` to avoid installing unused dependencies like Mocha and ESLint.

**Comment by denysvitali** [comment]
I'm sorry @CapacitorSet, I totally forgot this PR. I hope to find more time in the next weeks to review my code. Have a nice day 😊

**Comment by CapacitorSet** [comment]
Closing this for now, you're welcome to reopen this whenever you can fix it..

**Pull request closed without merging**